README.md

```diff
@@ -290,6 +290,23 @@ Use the `subscriptionCapability` subcommand to delete a capability from a subscr
 * Delete a capability from an organization `ocm support delete organizationCapability [orgID] [capabilityKey]`
 * Delete a capability from a subscription `ocm support delete subscriptionCapability [subscriptionID] [capabilityKey]`
 
+##### Deleting type independent capabilities
+
+Use the `capability` subcommand to delete a capability by passing the ID, or provide filter value to search matching capabilities and delete them. By default the dry run flag will be enabled. Pass `dryRun` flag to actually remove the resource.
```

Comment:
This bit is a little confusing: "Pass `dryRun` flag to actually remove the resource".

Something like, "Set dryRun flag to false to flag to actually remove the resource" would be more clear.